### PR TITLE
[sival] ROM_EXT SiVal updates.

### PR DESCRIPTION
**Pull request by moidx**
1. Update the info page lockdown configuration to include the `kFlashCtrlInfoPageFactoryId` page.
2. Updates the ROM_EXT SiVal documentation to include relevant details for this configuration.

### CODE REVIEW

File: sw/device/silicon_creator/rom_ext/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# ROM\_EXT
+
+The ROM\_EXT is the first boot mutable stage in the Secure Boot implementation,
```
Comment (timothytrippel) [inline sw/device/silicon_creator/rom_ext/README.md:3]:
```suggestion
The ROM\_EXT is the first (mutable) boot stage in the Secure Boot implementation,
```

File: sw/device/silicon_creator/rom_ext/doc/si_val.md [hunk 2]
```diff
@@ -37,21 +40,41 @@ mseccfg = 00000002                      ; RLB=0, MMWP=1, MML=0.
...
+signature verification of programs loaded into the flash owner partitions.
 
-TODO: Re-evaluate this if needed.
+Both the ePMP and SRAM controller will forbid execution from SRAM. The SRAM
+controller configuration is locked by the `ROM_EXT`.
 
 ### FLASH
 
-The FLASH info pages designeded for `silicon_creator` use will be locked as no-access.
+The FLASH info pages designeded for `silicon_creator` use are locked as
```
Comment (timothytrippel) [inline sw/device/silicon_creator/rom_ext/doc/si_val.md:56]:
```suggestion
The FLASH info pages designed for `silicon_creator` use are locked as
```

File: sw/device/silicon_creator/rom_ext/doc/si_val.md [hunk 3]
```diff
@@ -37,21 +40,41 @@ mseccfg = 00000002                      ; RLB=0, MMWP=1, MML=0.
...
+kFlashCtrlInfoPageCreatorSecret       | 0    | 1
+kFlashCtrlInfoPageOwnerSecret         | 0    | 2
+kFlashCtrlInfoPageWaferAuthSecret     | 0    | 3
+kFlashCtrlInfoPageAttestationKeySeeds | 0    | 4
+kFlashCtrlInfoPageBootData0           | 1    | 0
+kFlashCtrlInfoPageBootData1           | 1    | 1
+kFlashCtrlInfoPageOwnerSlot0          | 1    | 2
+kFlashCtrlInfoPageOwnerSlot1          | 1    | 3
+
+The canonical info flash reference used by the ROM and the ROM EXT is available
```
Comment (timothytrippel) [inline sw/device/silicon_creator/rom_ext/doc/si_val.md:71]:
```suggestion
The canonical info flash reference used by the ROM and the ROM\_EXT is available
```

**Review by timothytrippel — APPROVED** [review]

File: sw/device/silicon_creator/rom_ext/README.md (hunk 1 above)
Comment (moidx) [reply sw/device/silicon_creator/rom_ext/README.md]: Resolved.

File: sw/device/silicon_creator/rom_ext/doc/si_val.md (hunk 2 above)
Comment (moidx) [reply sw/device/silicon_creator/rom_ext/doc/si_val.md]: Done.

File: sw/device/silicon_creator/rom_ext/doc/si_val.md (hunk 3 above)
Comment (moidx) [reply sw/device/silicon_creator/rom_ext/doc/si_val.md]: Done.

**Pull request merged**